xlrd/book.py

```diff
@@ -1452,7 +1452,7 @@ def unpack_SST_table(datatab, nstrings):
                 rawstrg = data[pos:pos+2*charsavail]
                 # if DEBUG: print "SST U16: nchars=%d pos=%d rawstrg=%r" % (nchars, pos, rawstrg)
                 try:
-                    accstrg += unicode(rawstrg, "utf_16_le")
+                    accstrg += unicode(rawstrg, "utf_16_le", 'replace')
```

Comment:
How can such invalid encoding happen?

Is the whole file encoding wrong, or just a local corruption?

Reply:
There is high unicode codepoint which are represented as surrogate pairs in utf16. It turned out that python does not surrogates by default. So they need to be allowed explicitely. Must match to value encoding, like in https://github.com/RaRe-Technologies/pii_tools/pull/786

Reply:
What do you mean? The encoding here and in #786 does not match, `utf_16_le` vs `utf8`.

Reply:
Thats poorly written sentence. Encoding type itself can be arbitrary, but encoding process have to be configured in the same way (`surrogatepass`)